proto/il/v1/genesis.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package il.v1;
+
+import "gogoproto/gogo.proto";
+import "il/v1/il.proto";
+
+option go_package = "github.com/peggyjv/sommelier/x/il/types";
+
+
+// GenesisState all impermanent loss state that must be provided at genesis.
+message GenesisState {
+    Params params = 1 [(gogoproto.nullable) = false];
+    repeated StoplossPosition stoploss_positions = 2 [(gogoproto.nullable) = false];
+}
+
+// StoplossPosition represents an impermanent loss stop position for a given address and uniswap pair.
+message StoplossPosition {
```

Comment:
This looks good. Another way to do it would be to store them as `map<address>[]StopLoss`?

Reply:
we would need to iterate over the map on init genesis, so it could panic due to non-determinism.